contracts/tests/SpigotedLine.t.sol

```diff
@@ -937,5 +937,68 @@ contract SpigotedLineTest is Test {
     function test_can_whitelist_as_arbiter() public {
       assertTrue(line.updateWhitelist(bytes4("0000"), true));
     }
+
+    function test_cant_use_and_repay_if_unauthorized() public {
+      _borrow(line.ids(0), lentAmount);
+      vm.prank(address(1));
+      vm.expectRevert("Unauthorized");
```

Comment:
will need to update this once if/revert is implemented to `ILineOfCredit.CallerAccessDenied.selector`